sdk/node/src/utils/jsonldFrameWrapper.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * This function is invited to wrap json ld.frame function
+ * in case it returns not a valid segment but @context and @graph fields,
+ * under which in the first index segment with data is contained.
+ * This problem only related to vcs with new jsonld context.
+ * It is temporary solution until we will have time  to find,
+ * fix, get PR review in jsonld lib, merge it, publish,
+ * then  update jsonld lib in @mattrglobal/json-signatures-bbs,
+ * get PR review from them, merge it, publish and only then update this SDK
```

Comment:
So we cannot create our own jsonld (derived from the original one) because mattrglobal/json-signatures-bbs imports the original jsonld?

Reply:
Nope, we can use force resolution, but it will take much more time for implementation. In the same time, people already want to have this feature. What I see:
* add this monkey patch 
*  then resume work on finding the core of issue => go to jsonld and try to push fix here and in case of failing - create our own solution